applications/plugins/SofaPython/PythonScriptController.h

```diff
@@ -133,6 +135,7 @@ class SOFA_SOFAPYTHON_API PythonScriptController : public ScriptController
     PyObject *m_Func_onMouseWheel;
     PyObject *m_Func_onGUIEvent;
     PyObject *m_Func_onScriptEvent;
+    PyObject* m_Func_onEvent;
```

Comment:
What is this one for ? 
It adds a new onEvent, a Data<bool> and a hook in the onScriptEvent function. 
Shouldn't it be in a separated PR ?